packages/clay-css/src/scss/variables/_panels.scss

```diff
@@ -145,17 +172,30 @@ $panel-unstyled-header-link: () !default;
 $panel-unstyled: () !default;
 $panel-unstyled: map-deep-merge(
 	(
-		bg: transparent,
+		background-color: transparent,
 		border-radius: 0,
 		border-width: 0,
-		header-border-color: $gray-500,
-		header-border-style: solid,
-		header-border-width: 0 0 1px 0,
-		header-margin-bottom: 1rem,
-		header-collapsed-border-color: $gray-500,
+		margin-bottom: 1.5rem,
+		header: (
+			border-color: $gray-500,
+			border-radius: 0,
+			border-style: solid,
+			border-width: 0 0 1px 0,
+			padding-left: 0,
+		),
+		header-collapsed: (
+			border-color: $gray-500,
+		),
```

Comment:
I've noticed some patterns that we are trying to adopt with `map-deep`, this is more of a question, but why not add this inside the `header`, is this a nesting limitation?

Reply:
@matuzalemsteles no limitation. I'm not sure when I last touched the `clay-panel-variant` mixin. Patterns might have changed a bit since then. We could do:
```
header: (
    collapsed: (),
    c-inner: (),
    link: (),
)
```

Reply:
I would have to update the mixin, let me know if you want it done in a separate pr, otherwise I'll update it here.

Reply:
@pat270 thanks for the information, well if you think this is valid, we can go ahead with it, feel free to send it in this PR or another one, whichever is better.